assets/js/components/notifications/InternalServerError.js

```diff
@@ -21,15 +21,27 @@
  */
 import { useSelect } from 'googlesitekit-data';
 import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
-import BannerNotification from './BannerNotification';
+import NotificationError from '../../googlesitekit/notifications/components/layout/NotificationError';
+import Notification from '../../googlesitekit/notifications/components/Notification';
 
 export default function InternalServerError() {
 	const error = useSelect( ( select ) =>
 		select( CORE_SITE ).getInternalServerError()
 	);
+
 	if ( ! error ) {
 		return null;
 	}
 
-	return <BannerNotification { ...error } />;
+	return (
+		<Notification
+			skipOnViewTracking
```

Comment:
@zutigrm Why do we need to add this to purposely skip tracking? It is fine to have the default events added here. Even though the <InternalServerError> component is always rendered, the <Notification> component within is only rendered when the actual notification is displayed. So tracking the standard GA event is fine and works well in my testing.